src/accountant/core.cljs

```diff
@@ -83,7 +83,7 @@
            host (.getDomain uri)
            current-host js/window.location.hostname]
        (when (and (not any-key) (= button 0) (path-exists? path) (= host current-host))
-         (set-token! history relative-href title)
+         (. history (setToken relative-href title))
```

Comment:
I would also prefer `(.setToken...)` Where else are we using the style you're referring to?